src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/DefaultRazorConfigurationService.cs

```diff
@@ -63,13 +64,14 @@ public async override Task<RazorLSPOptions> GetLatestOptionsAsync(CancellationTo
 
                 var builder = new ConfigurationBuilder();
 
-                var razorJsonString = result[0].ToString();
-                using var razorStream = new MemoryStream(Encoding.UTF8.GetBytes(razorJsonString));
-                builder.AddJsonStream(razorStream);
-
-                var htmlJsonString = result[1].ToString();
-                using var htmlStream = new MemoryStream(Encoding.UTF8.GetBytes(htmlJsonString));
-                builder.AddJsonStream(htmlStream);
```

Comment:
The bug here was that I was putting both razor and html data in the same config. So when html added a `html->format->enable` option, it conflicted with the Razor one.